### ISSUE TITLE
Naming conventions in layout properties

### ISSUE THREAD

**Issue by gjimenezUCM** [plotly/plotly.py#1830]
It is not an issue exactly but more properly a suggestion about a confusing naming convention. 

In short, "paper" refers to 2 different concepts depending on the layout property (the plot container or the plotting area). 

If I want to change the background color of a plot I use the layout properties:

- [paper_bgcolor](https://plot.ly/python/reference/#layout-paper_bgcolor): Sets the color of **paper** where the graph is drawn.
- [plot_bgcolor](https://plot.ly/python/reference/#layout-plot_bgcolor): Sets the color of **plotting area in-between x and y axes**.

However, `xref`and `yref` properties from `title` can take two different values with some confusing names. [According to the documentation](https://plot.ly/python/reference/#layout-title-xref):

- "container" spans the entire `width` of the plot. (It seems that it refers to the "paper" described for paper_bgcolor).
- "paper" refers to the width of the plotting area only (and it corresponds to the "plot" in plot_bgcolor)

Some sugestions:

1. Maybe replacing:

- paper_bgcolor --> container_bgcolor
- plot_bgcolor --> paper_bgcolor

(However, it would be confusing with previous versions)

2. I think  that using paper and plot names for the xref/yref properties implies a deep refactoring in other layout elements, which use `paper` for naming the plotting area. 

Thanks for your work in Plotly.py!!

**Comment by emmanuelle** [comment]
You're right, there is some inconsistency in the way we use `paper`. As you say, because of backwards compatibility there is no easy fix. We could for example alias `paper_bgcolor` to `container_bgcolor` and promote `container_bgcolor` in the documentation, for less confusion (while keeping `plot_bgcolor` which is not confusing, and is used quite a lot). Or we could just document better the current behviour.

**Comment by gjimenezUCM** [comment]
Thanks for your fast reply!  I understand that there is no easy fix for this problem. I think that the solution that you proposes is less confusing than using `paper` for both container and plotting area.
Thanks again for your work in Plotly.py

**Comment by gvwilson** [comment]
Hi - we are currently trying to tidy up Plotly's public repositories to help us focus our efforts on things that will help users most. Since this issue has been sitting for several years, I'm going to close it. If it's still a concern, we'd be grateful if you could open a new issue (with a short reproducible example if appropriate) so that we can add it to our backlog. Thanks for your help - @gvwilson
